libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -724,6 +730,8 @@ DNNModel *ff_dnn_load_model_ov(const char *model_filename, DNNFunctionType func_
 
     //parse options
     av_opt_set_defaults(ctx);
+    // set default async to 1
+    ctx->options.async = 1;
```

Comment:
.i64=1 in dnn_backend_common.h  has set the default value as 1.

Reply:
Oh so this sets the default value. Okay I'll remove this line